\documentclass[10pt,oneside]{article}
\input{preamble}

\begin{document}

\title{Controlling Cas9 with ligand-activated and ligand-deactivated sgRNAs}
\author[1]{Kale Kundert}
\author[2]{James Lucas}
\author[3]{Kyle E. Watters}
\author[4]{Christina M. Fitzsimmons}
\author[3]{Benjamin L. Oakes}
\author[3]{Christof Fellman}
\author[2]{Andrew H. Ng}
\author[1]{Ben M. Heineike}
\author[3]{David F. Savage}
\author[3]{Jennifer A. Doudna}
\author[1]{Hana El-Samad}
\author[1]{Tanja Kortemme}

\affil[1]{Graduate Program in Biophysics, University of California San Francisco, San Francisco, California, USA}
\affil[2]{UCSF/UCB Graduate Program in Bioengineering, University of California San Francisco, San Francisco, California, USA}
\affil[3]{Department of Molecular and Cell Biology, University of California, Berkeley, Berkeley, California, USA}
\affil[4]{Chemistry and Chemical Biology Graduate Program, University of California San Francisco, San Francisco, California, USA}

\date{}

\maketitle{}

% A Brief Communication is a more concise format used typically to report a significant improvement to a tried-and-tested method, its modification and adaptation to an important original application, or an important new tool or resource of broad interest for the scientific community. This format typically does not exceed 3 printed pages. Brief Communications begin with a brief unreferenced abstract (3 sentences, no more than 70 words), which will appear on Medline. The title is limited to 10 words (or 90 characters). The main text is typically 1,000-1,500 words, including the abstract and contains no headings with the exception of a single heading for Methods to point readers to the online Methods section providing all technical details necessary for the independent reproduction of the methodology. Brief Communications normally have no more than 2 display items, although this may be flexible at the discretion of the editor, provided the page limit is observed. As a guideline, Brief Communications allow up to 20 references, and article titles are omitted from the reference list.

\section{Abstract}

% Brief Communications begin with a brief unreferenced abstract (3 sentences, no more than 70 words), which will appear on Medline. 

% Sentence 1: This is what we made.
% Sentence 2: This is what it can do.
% Sentence 3: This is why it's useful.

We present a new method for achieving small-molecule control of Cas9, based on the idea of using an aptamer to directly control whether or not the sgRNA can interact with Cas9.  Our sensors have a >10x dynamic range, are smoothly titratable, can respond to two different ligands, and can either be activated or deactivated by ligand.  We believe these sensors can be applied to probe the complex genetic interactions that pervade biology.

\section{Introduction}

%¶ Introduce Cas9, briefly review Cas9 engineering.

Ever since CRISPR/Cas9 emerged as a powerful system for engineering and studying biology, there has been broad interest in developing tools to better regulate its activity.  Such tools promise to help mitigate off-target effects and to allow the study of more complex biological perturbations \autocite{richter2017}.  To date, this problem has been approached by using a split-Cas9 construct \autocite{zetsche2015,nguyen2016,nihongaki2015}, incorporating unnatural caged amino acids into Cas9 \autocite{hemphill2015,luo2016}, fusing ligand-dependent localization signals to Cas9 \autocite{liuramli2016}, and inserting ligand- or light-sensitive domains into Cas9 \autocite{oakes2016,richter2016}.  These approaches, which focus on engineering Cas9 itself, are limited for applications involving gene expression (e.g. CRISPRi or CRISPRa) in that every gene being targeted will be regulated in the same way; there's no way to independently regulate different genes.

% Justify focusing on sgRNA rather than Cas9, briefly review sgRNA engineering, explain the advantages of our approach (Fig 1a).

% * <kortemme@cgl.ucsf.edu> 2017-12-07T22:25:25.333Z:
% 
% > Two other ligand-controlled sgRNA systems have been described recently
% mention prior work in the introduction instead and use it to motivate what we set out to do differently / better
% 
% ^ <kale@thekunderts.net> 2017-12-11T19:41:52.247Z:
% 
% I like this a lot better.
%
% ^.
By instead engineering Cas9's guide RNA (sgRNA), it becomes trivial to independently regulate different genes.  This approach has also been explored, but not as thoroughly.  One pair of methods is based on the idea of using an aptamer to control whether the sgRNA's \nt{20} target sequence (more succinctly called the spacer) is blocked by a complementary sequence installed elsewhere in the sgRNA \autocite{liu2016,tang2017}.  Another method is based on the idea of using the sgRNA to recruit protein regulators that need to be stabilized by a ligand to avoid degradation \autocite{maji2017}.  In contrast, our goal in this study was to engineer ligand-responsive sgRNAs by using an aptamer to directly affect the interaction between the sgRNA and Cas9 (\reffig{1}a).  There are several advantages to this approach.  First, it doesn't require any host-cell machinery or exogenous protein parts (other than Cas9).  Second, because it doesn't require any complementarity to the spacer, it is compatible with the cloning strategies needed to make pooled sgRNA libraries \autocite{gilbert2014}.  Third, because it is compatible with library screening, we could optimize the dynamic range of our sensors by using fluorescence-activated cell sorting (FACS).  And fourth, we could use the same approach to produce both ligand-activated and ligand-deactivated sensors.

\section{Results}

% The results section needs to convince the reader that the central claim is supported by data and logic. Every scientific argument has its own particular logical structure, which dictates the sequence in which its elements should be presented.

% Each paragraph in the results section starts with a sentence or two that sets up the question that the paragraph answers. The middle of the paragraph presents data and logic that pertain to the question, and the paragraph ends with a sentence that answers the question.

% Central claim: We can control CRISPRi with ligand-activated and ligand-deactivated sgRNAs.

%¶ All three solvent-exposed stems can be used to control the sgRNA (Fig 1b,1d,1e, Supp Tab [rational designs]).  Not enough to simply replace the stem.

We began by screening 97 rational designs using an \invitro{} Cas9 cleavage assay (\refsupptab{rational-designs}) to learn which insertion sites were most sensitive to the presence of the theophylline aptamer \autocite{jenison1994}, and which strategies for linking the aptamer to the sgRNA were most effective.  Regarding insertion sites, we tried inserting the aptamer into each of the stems in \reffig{1}b, which are all solvent-exposed in the crystal structure of Cas9 in complex with sgRNA and DNA \autocite{nishimasu2014}.  Regarding linking strategies, we tried replacing different parts of each stem with the aptamer, splitting the sgRNA in half and using the aptamer to bring the halves together, and designing specific strand displacements for the \apo{} and \holo{} states.  We found functional sensors for all three insertion sites, and were most successful with the strand displacement strategy (\reffig{1}d, \refsupptab{rational-designs}).  All of these sensors could be titrated (\reffig{1}e).  We also unexpectedly learned that inserting the aptamer into the nexus region yields sensors that respond to the absence of theophylline, rather than to the presence of it.  Knowing how to design both ligand-activated and ligand-deactivated sgRNAs \invitro{}, we next sought to find sensors that would function robustly in \ecoli{}.
% * <kortemme@cgl.ucsf.edu> 2017-12-07T20:29:49.573Z:
% 
% > Escherichia coli
% I think "in vivo" means animals for most biologists....  Let's be specific
% 
% ^.
% * <kortemme@cgl.ucsf.edu> 2017-12-07T20:27:07.592Z:
% 
% > All of these sensors could be titrated 
% All?  make more specific which were tested
% 
% ^.
% * <kortemme@cgl.ucsf.edu> 2017-12-07T20:23:58.178Z:
% 
% > We began by screening 97 rational designs 
% separate first section of the results into two paragaphs: 
% 1) design strategy (Fig 1 current panels a-c) - this  is a key contribution of the paper!
% 2) in vitro cleavage assay results
% 
% ^.

%¶ We did a FACS screen to find linker sequences that would work well (Fig 2a,2b,2c, Supp Fig [in vitro in vivo]).

We used FACS screens based on the repression of GFP and RFP via CRISPRi to optimize our rational designs \invivo{}.  The strongest rational designs exhibited only weak activity in the CRISPRi assay (\refsuppfig{in-vitro-in-vivo}).  Since we previously found the base-pairing of the linker sequence to be important, we designed libraries with randomized linkers for all three insertion sites \refsupptab{libraries}.  For each library, we performed the four screens outlined in \reffig{2}a.  The first two screens select and counter-select for activity in the presence (or absence) of theophylline.  The second two screens do the same, but in the context of a different spacer.  Swapping the spacer is a crucial step.  Without it, most of the sensors we isolated were specific for the spacer used in the first screens.  The most robust ligand-activated sensor (\ligrnaF{}; i.e. sgRNA that's active in the + ligand state) and ligand-deactivated sensor (\ligrnaB{}; i.e. sgRNA that's active in the − ligand state) that we isolated are shown in \reffig{2}b,c.  With a spacer that was not used in any of the screens, both sensors have a >10x dynamic range and negligible overlap between the active and inactive populations (\reffig{2}b,c).
% * <kortemme@cgl.ucsf.edu> 2017-12-07T20:31:01.304Z:
% 
% > Since we previously found the base-pairing of the linker sequence to be importan
% not clear from previous section - highlight more specifically when explaining the rational design strategy
% 
% ^.

%¶ Our designs are spacer-independent (Fig 2d, Supp Fig [spacer-gel], [affinity-correlation]).
% * <kale@thekunderts.net> 2017-12-03T08:40:25.474Z:
% 
% I kinda want to remove the reference to {spacer-gel}, i.e. Fig S2.  I think it distracts from the point without really adding anything.
% 
% ^ <kortemme@cgl.ucsf.edu> 2017-12-07T20:35:15.645Z:
% 
% yes - would do this:
% "... Cas 9 cleavage assay (Figure 2d, Figure S2, Table S4)." and delete "One of the gels for the top-left spacer is shown for perspective (\refsuppfig{spacer-gel})"
% 
% ^ <kale@thekunderts.net> 2017-12-09T00:29:07.997Z:
% 
% Ok, done.
%
% ^.
Because RNA devices are known to be influenced by the sequences surrounding them \autocite{liang2012}, we tested \ligrnaF{} and \ligrnaB{} in the context of 24 different spacers using our \invitro{} Cas9 cleavage assay (\reffig{2}d, \refsuppfig{spacer-gel}, \refsupptab{spacer-assay}).  Both sensors respond to theophylline for the majority of the tested spacers.  We predicted the affinity between each spacer and the aptamer (plus its associated linker) for both ligRNAs to test the hypothesis that base-pairing with the aptamer could explain why the minority of spacers are detrimental to function (\refsuppfig{affinity-correlation}).  For \ligrnaF{} the correlation is negligible, but for the \ligrnaB{} it seems prudent to choose spacers with weak affinity for the aptamer, preferably weaker than \kcalmol{-15}.  These results suggest that ligRNAs should be capable of regulating most genes, especially those that can be targeted by multiple spacers.

%¶ Our designs exhibit a remarkably graded response to theophylline  (Fig 2e).  Inflection point is ...  Smooth response, wide range, maybe useful for tuning gene expression.  EC50 much greater than theophylline Kd.

% The Carothers2010 reference doesn't quite make the point I'm trying to make.  I should keep looking to see if I can find a paper that deals with the difference between Kd and EC50 more directly.

Next, we tested the ligRNAs with decreasing concentrations of theophylline (\reffig{2}e).  We observed that both sensors were smoothly titratable and exhibited a nearly linear response over a 50x range of ligand concentration.  One thing to note is that the apparent EC50 of the ligRNAs (about \uM{280}) is much higher than the $\mathsf{K_D}$ of the theophylline aptamer alone (\nM{320}) \autocite{jenison1994}.  This discrepancy is common for RNA devices, and could be explained by the magnesium concentrations in the cytoplasm \autocite{carothers2010} and the extra cost associated with refolding the device in addition to the aptamer (and possibly assembling the Cas9 complex, in this case).  Regardless, the titration confirms that the ligRNAs respond to theophylline and demonstrates that they may be useful for not just turning genes on or off, but also for precisely controlling their levels of expression.
% * <kortemme@cgl.ucsf.edu> 2017-12-07T20:36:54.090Z:
% 
% > could be explained by the conditions in the cytoplasm 
% make less vague
% 
% ^ <kale@thekunderts.net> 2017-12-09T00:28:07.442Z:
% 
% Ok, really what I meant was magnesium concentrations.
% 
% ^.

%¶ Our designs are not sensitive to the expression level of the sgRNA (Supp Fig [j23150]).

We also tested how decreased expression levels would affect the ligRNAs by replacing the strong constitutive promoter used in every experiment up to this point (J23119) with a weak constitutive promoter (J23150) and repeating the CRISPRi assay (\refsuppfig{j23150}).  Both ligRNAs remained functional with both promoters.  In addition, the weaker promoter shifted the dynamic ranges of both ligRNAs in the direction of increased gene expression, to the point where nearly full gene activation was achieved.  These data provide evidence that the ligRNAs are not unduly sensitive to their environment, and the ability to tune the dynamic range via promoter strength may be important for applications that require either full gene activation or full gene repression.

%¶ We sought to understand the mechanism by which our ligRNAs functioned. (Supp Fig [vienna predictions], [mutagenesis], [shape-seq])

% I decided to keep the narrative simpler by leaving the mutants we don't understand out of the main text.  I'll talk about them in the caption of the supplementary figure, instead.

% On a second reading, I'm not sure if this is a good idea.  The case for our mechanism doesn't really sounds that strong, especially for how long I talk about it...

We sought to understand the mechanism by which ligRNAs are activated and deactivated by doing mutagenesis experiments motivated by \insilico{} secondary structure predictions.  We used ViennaRNA \autocite{lorenz2011} to make predictions, and we modeled the ligand-bound state by adding a \kcalmol{-9.21} bonus for closing the ends of the aptamer.  For \ligrnaF{}, we predicted that the sgRNA scaffold was misfolded in the absence of ligand due to 6 nucleotides in the aptamer base-pairing with 6 nucleotides in the nexus (\refsuppfig{vienna-predictions}).  Unfortunately, testing this hypothesis via mutagenesis would've required mutating the theophylline binding motif of the aptamer, and we did not pursue this mechanism further.  For \ligrnaB{}, there was no difference in the predicted structures for the \apo{} and \holo{} states (\refsuppfig{vienna-predictions}).  However, we noticed that the stem installed by our screen retained a uracil that --- in the sgRNA scaffold --- flips out and buries itself in Cas9 \autocite{nishimasu2014}.  Because this uracil makes a wobble base pair in a stem leading up to the aptamer, we hypothesized that the stem unzips enough to allow the uracil out when ligand is absent, but zips closed and prevents Cas9 binding by sequestering the uracil when ligand is present (\refsuppfig{mutagenesis}a).  We began testing this hypothesis by moving the uracil to the opposite strand while keeping the wobble base-pair intact, and this rendered \ligrnaB{} completely inactive (\refsuppfig{mutagenesis}d).  We continued by modulating the strength of the base-pairs between the uracil and the aptamer, and consistent with the idea that the stem is being zipped and unzipped, we found that weaker base pairs were more repressing while stronger base pairs were more activating (\refsuppfig{mutagenesis}e).  We conclude that this mechanistic insight explains how an aptamer could control the assembly of a Cas9 complex and suggests more ways for ligRNAs to be tuned for specific applications.

%¶ ligRNA⁻ can be used to control two genes with two different ligands (Fig 3, Supp Fig [ligand-matrix]).

Finally, we tested whether ligRNAs could be used to simultaneously control two different genes with two different ligands.  The two ligands we used were theophylline (theo) and 3-methylxanthine (3mx).  The corresponding aptamers differ in only one position, so we simply replaced one aptamer with the other without further optimization.  \ligrnaF was negatively affected by this naive approach (\refsuppfig{ligand-matrix}) but \ligrnaB was not, so we used only the latter to construct a system that expresses GFP upon addition of theophylline and expresses GFP and RFP upon addition of both ligands (\reffig{3}a,b).  We then performed a timecourse where we sequentially activated, deactivated, and reactivated both reporter genes using ligRNAs (\reffig{3}c).  The caveat with this particular ligand pair is that the aptamers are not orthogonal because the theophylline aptamer is triggered by both ligands.  Despite this, we have demonstrated that ligRNAs can be used to sequentially activate two different genes.
% * <kortemme@cgl.ucsf.edu> 2017-12-07T20:41:40.766Z:
% 
% > we've
% no contractions...
% 
% ^ <kale@thekunderts.net> 2017-12-09T00:27:15.283Z:
% 
% ok. :-)
%
% ^.
% * <kortemme@cgl.ucsf.edu> 2017-12-07T20:40:52.537Z:
% 
% > Finally, we tested whether ligRNAs could be used to simultaneously control two different genes with two different ligands
% highlight importance - this experiment is key!
% 
% ^.

\section{Discussion}

% The discussion section explains how the results have filled the gap that was identified in the introduction, provides caveats to the interpretation, and describes how the paper advances the field by providing new opportunities.  This is typically done by recapitulating the results, discussing the limitations, and then revealing how the central contribution may catalyze future progress.

%¶ Our ligRNAs are not functional in eukaryotic systems (Supp Fig [mammalian data], [yeast data]) but are still useful in bacteria.
% * <kale@thekunderts.net> 2017-12-11T21:53:37.073Z:
% 
% I rewrote the second half of this paragraph to focus on potential applications in bacteria.  I think it helps a lot to end the paper on a more optimistic note.  The question is whether I should mention that the Liu constructs didn't work in bacteria in our hands.  I left it out for now; I felt the paragraph was strong enough without it.  But it's something to think about.
% 
% ^.
Although ligRNAs function robustly in bacteria, we were unable to successfully transfer them to eukaryotic systems.  We first tried a gene-editing assay in a HEK293-based cell line, and observed no editing for any of the ligRNAs at any non-lethal concentration of theophylline (\refsuppfig{mammalian-data}).  We then tried a gene-repression assay in yeast strain BY4741, and observed no repression for \ligrnaF{} and depending on the spacer sequence either no repression or full repression for \ligrnaB{} (\refsuppfig{yeast-data}).  Optimizing the ligRNAs for eukaryotic systems is a promising direction for future work.  Regardless, ligRNAs have the potential to be very useful tools even if they only ever function in bacteria.  First, many interesting (e.g. pathogenic or symbiotic) species of bacteria do not have facile genetic controls available, and ligRNAs provide such controls with a very minimal footprint.  Second, temporal interactions between genes are known to be important for disease \autocite{lee2012}, and ligRNAs provide a way to conduct large-scale screens for similar effects in the interactions between bacteria and their environments \autocite{peters2016}.
% * <kortemme@cgl.ucsf.edu> 2017-12-07T22:29:38.989Z:
% 
% >  We then tried a gene-repression assay in yeast strain BY4741, and observed no repression for \ligrnaF{} and either no repression or full repression for \ligrnaB{} depending on the spacer sequence (\refsuppfig{yeast-data}). 
% Is this from Ben's and Andrew's data?  Have they (or you) repeated these experiments to confirm?
% 
% ^ <kale@thekunderts.net> 2017-12-11T22:39:22.520Z:
% 
% It's their data.  I never repeated these experiments myself.  I sent them an email asking for clarification.
%
% ^.

%¶ We believe that it will be increasingly important to study the spatial and temporal dependencies on genetic circuits, and believe that high-throughput way to regulate gene expression will be critical in the endeavor.

We believe that the study of subtle and complex biological systems will increasingly require the ability not just to perturb individual genes, or to knock down many different sets of genes, but to gently perturb many different genes at the same time.  ligRNAs provide this capability by putting a fine-control knob on the already powerful CRISPRi technology.

\section{Methods}

\subsection{\Invitro{} Cas9 cleavage assay}

\Invitro{} transcription: Linear, double-stranded template DNA was acquired either by ordering gBlocks® Gene Fragments from IDT (\reffig{1}d,e) or by cloning the desired sequence into a pUC vector and digesting it with with EcoRI and HindIII (\reffig{2}d).  Each construct had the following T7 promoter --- \seq{TATAGTAATAATACGACTCACTATAG} --- and a spacer sequence that began with at least 3 G's.  The HiScribe™ T7 High Yield RNA Synthesis Kit (NEB E2040S) was used to transcribe \ngx{10-50} of DNA template, and the RNA Clean \& Con\-cen\-trator™-25 spin columns (Zymo R1018) were used to remove unincorporated nucleotides.  Target DNA: Target DNA was prepared by using inverse PCR to clone the appropriate target sequence into the pCR2.1 vector roughly across from its XmnI site (i.e.\ in the MCS).  The vector was then digested with XmnI (NEB R0194S) as follows: mix \uL{43.5} \nguL{≈500} miniprepped pCR2.1 DNA, \uL{5.0} 10x CutSmart buffer, and \uL{1.5} \UuL{20} XmnI; incubate at 37°C until no uncleaved plasmid is detectable on a gel (usually \minute{30--60}); dilute to \nM{30}; store at -20°C.  Cas9 reaction: We adapted the following protocol from \namedcite{briner2014}: mix \uL{5.0} water or \mM{30} theophylline and \uL{1.5} \uM{1.5} sgRNA; incubate at 95°C for \minute{3}, then at 4°C for \minute{1}; add \uL{5.48} water, \uL{1.5} 10x Cas9 buffer, and \uL{0.02} \uM{20} Cas9 (NEB M0386T) via master mix; incubate at room temperature for \minute{10}; add \uL{1.5} \nM{30} target DNA; pipet to mix; incubate at 37°C for \hour{1}; add \uL{0.09} \mgmL{20} RNase~A (Sigma R6148), \uL{0.09} \mgmL{20} Proteinase~K (Denville CB3210-5), and \uL{2.82} 6x loading dye via master mix; incubate at 37°C for \minute{20}, then at 55°C for \minute{20}; run the entire reaction (\uL{18}) on a 1\% agarose/TAE/GelRed gel at \Vcm{4.5} for \minute{70}.  Gel quantification: Band intensities were quantified using Fiji (1.51r).  The background was subtracted from each image using a \px{50} rolling ball radius.  The fraction of DNA cleaved in each lane was calculated:

\begin{displaymath}
 \mathrm{f} = \frac{\mathrm{pixels}_\mathrm{2kb}}{\mathrm{pixels}_\mathrm{4kb} + \mathrm{pixels}_\mathrm{2kb}}
\end{displaymath}

The change in cleavage due to ligand was calculated: 

\begin{displaymath}
 \mathrm{Δf} = \mathrm{f}_\mathrm{theo} - \mathrm{f}_\mathrm{apo}
\end{displaymath}

\subsection{CRISPRi assay}

Strain: The strain used for all CRISPRi experiments was \ecoli{} MG1655 with dCas9 and ChlorR on a p15A plasmid (pgRNA-bacteria, Addgene 44251), sgRNA and AmpR on a pUC plasmid (pdCas9-bacteria, Addgene 44249), and sfGFP \autocite{pedelacq2006}, mRFP \autocite{campbell2002}, and KanR chromosomally integrated.  This strain was originally described by \namedcite{qi2013}

Flow cytometry: Overnight cultures were started from freshly picked colonies in \mL{1} LB with \ugmL{100} carbenicillin and \ugmL{35} chloramphenicol.  The next morning, day cultures were started in \mL{15} culture tubes or 24-well blocks by inoculating \uL{4} of overnight culture into \mL{1} EZ Rich Defined Medium (Teknova M2105) with 0.1\% glucose, \ugmL{1} an\-hydro\-tetra\-cycline, \ugmL{100} carbenicillin, \ugmL{35} chloramphenicol, and either \mM{1} theophylline or not.  These cultures were then grown for 8h at 37°C with shaking at \rpm{225} before GFP and RFP fluorescence were measured using a BD LSRII flow cytometer.  Approximately \unit[10,000]{events} were recorded for each measurement.  Biological replicates were done on different days using different colonies from the same transformation.

Data analysis: Cell distributions were obtained by computing a Gaussian kernel density estimation (KDE) over the base-10 logarithms of the measured fluorescence values.  The mode was considered to be the center of each distribution (e.g.\ for determining fold changes) and was obtained through the BFGS maximization of the KDE.

\subsection{FACS screen}

Library cloning: Randomized regions were inserted into the sgRNA using inverse PCR with phosphate-modified and HPLC-purified primers containing degenerate nucleotides.  The PCR and ligation reactions were setup as follows: mix \uL{19.0} water, \uL{2.5} \uM{5} forward primer, \uL{2.5} \uM{5} reverse primer, \uL{1.0} \pguL{100} template DNA and \uL{25.0} Q5® High-Fidelity 2x Master Mix (NEB M0492L); run PCR according to NEB's recommendations, with a \minute{2} extension time and an annealing temperature around 60°C; add \uL{1} \UuL{20} DpnI (NEB R0176L); incubate at 37°C for \hour{1}; purify using QIAquick spin columns (Qiagen 28106) and elute in \uL{50} water; add \uL{5.67} 10x T4 ligase buffer and \uL{1} \UuL{400} T4 ligase (NEB M0202L); incubate overnight at 16°C; load the ligated DNA onto Zymoclean™ spin columns (Zymo D4002) and elute in \uL{10} water to desalt and concentrate it.

Electrotransformation: Electrocompetent cells were prepared as follows: make ``low-salt'' SOB media: \g{20} bacto-tryptone, \g{5} bacto-yeast extract, \mL{2} \molar{5} NaCl, \uL{833.3} \molar{3} KCl, water to 1L, pH to 7.0 with NaOH, autoclave 30 min at 121°C; pick a fresh colony and grow overnight in \mL{1} SOB; in the morning, inoculate \liter{1} SOB with the entire overnight culture; grow at 37°C with shaking at \rpm{225} until OD=0.4 (\hour{≈4}); place cells in an ice bath for 10 min to quickly stop their growth; wash with 400 mL pre-chilled water, then 200 mL pre-chilled water, then 200 mL pre-chilled 10\% glycerol; resuspend in a total volume of \mL{6} pre-chilled 10\% glycerol; make \uL{100} aliquots; flash-freeze and store at -80°C.  Electrocompetent cells were transformed as follows: thaw competent cells on ice for \minute{10}; pipet once to mix cells with \uL{2} \nguL{≈250} library plasmid; shock at \kV{1.8} with a \ms{5} decay time; immediately add \mL{1} pre-warmed SOC; recover at 37°C for \hour{1}; dilute into selective liquid media and grow at 37°C with shaking at \rpm{225} overnight.  After PCR and ligation, libraries were first transformed into electrocompetent Top10 cells, then mira-prepped \autocite{pronobis2016}, sequenced, and combined as necessary, then transformed again into electrocompetent MG1655 cells already harboring the dCas9 plasmid.

Sorting: Cells were grown as for the CRISPRi assay, but when starting new  cultures, care was taken to subculture at least 10x more cells than the size of the library (often \uL{200}).  Sorting was done using a BD FACSAria II cell sorter.  Sorting was no slower than \evts{1000} and no faster than \evts{20,000}, with the slower speeds being more accurate and the faster speeds being necessary to sort large libraries.  Gates were drawn based on the position of the wildtype population if possible, and based on the most extreme library members otherwise.  All gates were drawn diagonally in GFP vs.\ RFP space.  Sorted cells were collected in \mL{1} SOC and, after sorting, were diluted into selective media and grown at 37°C with shaking at \rpm{225} overnight.

Screening for \ligrnaB{}: Libraries are as in \refsupptab{libraries}

\subsection{Spacer assay}

Choice of spacers: The spacers for this assay were chosen by a script that generated uniformly random sequences, scored them by the method of \namedcite{doench2016}, and kept only those that scored higher than 0.5 (the median).  This approach was designed to produce spacers that were as unbiased as possible, while still being likely to work well with the controls.  In fact, the average cleavage for the positive controls was 97\%, and the lowest cleavage for any of the positive controls was 78\% (\refsupptab{spacer-assay}).

\subsection{RNA secondary structure predictions}

Secondary structure predictions were made using the RNAfold program from the ViennaRNA package (version 2.4.3).  The structures reported here are minimum free energy (MFE) predictions, although centroid and maximum expected accuracy (MEA) structures from partition function calculations were nearly identical in every case.  The \holo{} state was simulated using soft constraints: a \kcalmol{-9.21} bonus was granted for forming the base pair flanking the aptamer.  This bonus corresponds to the \nM{320} affinity of the theophylline aptamer for its ligand \autocite{jenison1994}:

\begin{align*}
\Delta G
    &= RT \ln K_D \\
    &= (\unitfrac[\mathrm{1.987 \times 10^{-3}}]{kcal}{mol \cdot K}) 
       (\unit[\mathrm{310}]{K}) 
       \ln (\unit[\mathrm{3.20 \times 10^{-4}}]{M}) \\
    &= \unitfrac[\mathrm{-9.213}]{kcal}{mol} \\
\end{align*}

The command-lines used for the \apo{} and \holo{} states, respectively, are given below:

\begin{verbatim}
$ RNAfold --partfunc --MEA
$ RNAfold --partfunc --MEA --motif \
    "GAUACCAGCCGAAAGGCCCUUGGCAGC,(...((((((....)))...)))...),-9.212741321099747"
\end{verbatim}



\section{Abbreviations}

\begin{itemize}
    \item CRISPR: Clustered regularly interspaced short palindromic repeats
    \item Cas9: CRISPR-associated protein 9
    \item CRISPRi: CRISPR-inhibition
    \item CRISPRa: CRISPR-activation
    \item FACS: Fluorescence activated cell sorting
    \item GFP: Green fluorescent protein
    \item RFP: Red fluorescent protein
\end{itemize}

\section{Contributions}

\begin{itemize}
    \item KK conceived the project, performed most of the experiments, and wrote the paper.
    \item JL performed many of the CRISPRi experiments.
    \item KEW proposed the mechanism for \ligrnaB{} and performed the SHAPE-Seq experiments.
    \item CMF advised on the \invitro{} Cas9 experiments.
    \item BLO advised on the FACS screens.
    \item CF tested the ligRNAs in mammalian cells.
    \item AHN and BMH tested the ligRNAs in yeast cells.
    \item DFS, JAD, HES, and TK provided mentorship.
\end{itemize}

\section{Acknowledgments}

The authors would like to thank Anna Simon for presenting the ideas that motivated this project, Kianna Zucker for cloning most of the constructs used in \reffig{2}d, and Kyle Barlow and Anum Glasgow for helping with miscellaneous experiments.

\section{Figures}

\fig{1}

 \figonecol{figure_1}

 \captitle{Rational design of ligand-sensitive sgRNAs}
 (a) Schematic illustrating the design strategy.
% * <kortemme@cgl.ucsf.edu> 2017-11-02T12:46:38.023Z:
%
% more details here after Fig revision
%
% ^.
 (b) Aptamer insertion sites \autocite{briner2014}.
% * <kortemme@cgl.ucsf.edu> 2017-11-02T12:47:34.000Z:
%
% insert ref after "domains"
%
% ^.
 (c) Ligand binding causes the ends of the theophylline aptamer to coalesce into a rigid, stem-like structure \autocite{zimmerman1997}.
% * <kortemme@cgl.ucsf.edu> 2017-11-02T12:48:01.000Z:
%
% delete panel c
%
% ^.
 (d) Efficiency of \invitro{} Cas9 cleavage of DNA in the presence and absence of 10mM theophylline for selected ligRNAs.  Design numbers refer to \refsupptab{rational-designs} and are color-coded by aptamer insertion sites.  Percent cut values (bottom) are the average of at least two experiments.  All data shown are from a single gel.
% * <kortemme@cgl.ucsf.edu> 2017-11-02T12:56:29.000Z:
%
% should replace "sgRNA construct" with whatever nomenclature we decide on, i.e. ligRNA- etc
%
% ^.
 (e) Graded cleavage efficiency in response to increasing concentrations of theophylline for a selected design.

\fig{2}

 \figtwocol{figure_2}

 \captitle{Robust ligRNAs identified by \invivo{} screen}
 %
 (a) A schematic of the process used to isolate \ligrnaF{}.  The numbers above the arrows indicate approximate library sizes at each step.  G1, G2, R1, and R2 refer to the first and second spacers targeting GFP and RFP, respectively.
 (b,c) Single-cell fluorescence distributions for \ligrnaF{} (teal) and \ligrnaB{} (navy) with (solid lines) and without (dashed lines) theophylline.  Control distributions are in grey.  The mode of each distribution is indicated with a plus sign.  Fluorescence values for each cell are normalized by GFP fluorescence for that cell and the modes of the un-repressed control populations measured for that replicate.
 (d) \Invitro{} cleavage data for both ligRNAs in the context of 24 randomly chosen spacers.  The colors indicate the change in the percent of DNA cleaved with and without theophylline.  Each box represents the mean of three measurements with a single spacer.   The grey error bar in the color scale shows the mean and standard deviation for 143 control measurements.
 (e) \Invivo{} theophylline titration for both ligRNAs.  The fluorescence axis is the same as in (b) and (c).  The fits are to a logistic curve.
 

\fig{3}

 \figonecol{figure_3}

 \captitle{Parallel control of two genes with two ligands} 
 (a,b) Schematic illustrating the constructs used in the experiment, and the expected consequences of adding theophylline (theo) and 3-methylxanthine (3mx) for each reporter.
 (c) Fluorescence values measured over a \hour{48} timecourse.  Single-cell fluorescence values were measured by flow cytometry.  Bar heights and error bars represent the modes and standard deviations of the cell distributions, respectively.  Unlike in \reffig{2}, fluorescence is normalized by side-scatter (SSC) because both fluorescent channels are being manipulated.

\section{Supplementary Materials}

\supptab{components}

    \tabtwocol{supp_mat/components/components_tabular}

    \captitle{Miscellaneous DNA sequences used in this project.}

\supptab{libraries}

    \tabtwocol{supp_mat/library_sequences/library_sequences_tabular}

    \captitle{Library sequences screened via FACS in this project.}  \ligrnaF{} came from library \#29, while \ligrnaB{} came from library \#22.  Libraries \#29 and \#30 are based on the best clone isolated from library \#26.

\supptab{rational-designs}

    \tabtwocol{supp_mat/rational_design_sequences}

    \captitle{Complete results from the \invitro{} screen of rational designs.}  
    \#: The number used to refer to a rational design in the main text.
    Strategy: The mechanism by which the design was intended to work.  "Stem Replacement" means a stem in the sgRNA was simply replaced with by the aptamer (possibly with a linker).  "Induced dimerization" means the sgRNA was slit in half, with each half getting part of the aptamer, in the hope that the two halves would dimerize in the presence of ligand.  "Strand displacement" means that we designed strands that could base pair in two ways --- one with wildtype sgRNA 2° structure and the other somehow different --- in the hope that the aptamer would trigger a switch between the two conformations.
    Domain: Where in the sgRNA the aptamer was inserted.
    Cleavage: The percent of DNA that was cleaved by a design in the \invitro{} assay.  The \apo{} and \holo{} columns refer to the cleavage with and without theophylline, respectively, the Δ column is the difference between those, and the σ column is the standard deviation of the Δ values for the designs with more than one replicate.  All the percentages are the average of any replicates and are rounded to the nearest 1.
    N: The number of replicates for each design.
    Active: We considered a design to be active if it exhibited a >15\% change in cleavage in response to ligand.  
% * <kortemme@cgl.ucsf.edu> 2017-12-07T20:10:33.659Z:
% 
% > Active
% I don't think we can call a design "active" if the standard deviation is larger than the signal -  as for 2 of the stem replacement designs (i.e. 15+/- 21% etc).  Actually, looking at this in more detail, should only list standard deviation for N=>3
% 
% ^.
    Sequence: The sequence of the design, including the spacer AAVS spacer used in this assay.

\supptab{spacer-assay}

    \tabtwocol{supp_mat/spacer_assay}

    \captitle{Complete results from the \invitro{} spacer-dependence assay.}
    \#: The row and column in the heatmap in \reffig{2}d corresponding to this spacer.
    Sequence: The \bp{30} target sequence that was present in the DNA.  In upper case is the \bp{20} spacer sequence that was present in the sgRNA.  Note that each spacer begins with 3 \seq{G}'s, which is necessary for good transcription by T7 polymerase.  The \seq{NGG} PAM is just to the right of the sgRNA spacer sequence.  The way in which these sequences were chosen is described in the Methods section.
    Score: The score of the target sequence by the method of \namedcite{doench2016}.  Sequences with scores lower than 0.5 (about half of those generated) were not used.
    pos, neg, \ligrnaF{}, \ligrnaB{}: Cleavage data for the indicated sgRNA.  See \refsupptab{rational-designs} for the meaning of the \apo{}, \holo{}, Δ, σ, N columns.

\suppfig{in-vitro-in-vivo}

    \fignatsize{supp_mat/in_vitro_in_vivo}

    \captitle{The strongest rational designs have weak ligand-sensitivity \invivo{}}  Flow cytometry traces and fold change plots for the rational designs that were tested \invivo{}.  The labels on the y-axis refer to \refsupptab{rational-designs}.  Traces are color-coded by where the aptamer was inserted.  The x-axis is RFP fluorescence normalized by GFP fluorescence and the negative control.  All other lines and symbols as described in \reffig{2}.  Data are from three experiments performed on the same day (in all other experiments reported in this paper, replicates were performed on different days).

\suppfig{spacer-gel}

    \fignatsize{supp_mat/spacer_gel}

    \captitle{Representative gel from the \invitro{} spacer assay.}  This data is a single replicate for spacer \#1,1 (i.e. the top left corner of the \reffig{2}d).  The upper and lower bands are uncleaved and cleaved DNA, respectively.  Each design was tested in the absence and presence of theophylline.  The amount of DNA cleavage was quantified by gel densiometry and is reported as a percentage below each lane.

\suppfig{affinity-correlation}

    \fignatsize{supp_mat/affinity_correlation}

    \captitle{Correlation between ligRNA function and the amount of base-pairing between the spacer and the aptamer insert} Binding affinities (y-axis) were calculated using the \code{duplexfold} method from the python3 API of the ViennaRNA package (version 2.4.3).  This method returns the minimum binding energy between two strand of RNA considering only inter-strand base pairs.  For each calculation, the first strand was one of the 24 \nt{20} spacer sequences used in the \invitro{} spacer assay (\refsupptab{spacer-assay}).  The second strand was \seq{GCCGAUACCAGCCGAAAGGCCCUUGGCAGCGAC} for \ligrnaF{} or \seq{GUGGGAUACCAGCCGAAAGGCCCUUGGCAGCCUAC} for \ligrnaB{}.  These sequences include both the aptamer and the randomized linker connecting the aptamer to the rest of the sgRNA scaffold.  Percent cleavage values (x-axis) are the means of three replicates from the \invitro{} spacer assay (\reffig{2}d).  Linear regressions (solid lines) and R-values are shown.

\suppfig{j23150}

    \fignatsize{supp_mat/j23150}

    \captitle{Sensor activity is predictably modulated by promoter strength.}  Flow cytometry traces and fold change plots for three ligRNAs (the two discussed in the main text and a third "alternate" isolated from the same screen as \ligrnaF{}) in the context of strong (J23119) and weak (J23150) constitutive promoters.  All lines and symbols are described in \reffig{2}.  The alternate ligRNA actually has a larger dynamic range with the weaker promoter, and may be a useful for applications where lower concentrations of ligRNA are anticipated.

\suppfig{vienna-predictions}

    \figtwocol{supp_mat/vienna_predictions/vienna_predictions.pdf}

    \captitle{Secondary structure predictions suggest mechanisms of ligand sensitivity} The sequences of \ligrnaF{} (a) and \ligrnaB{} (b) are color-coded by domain.  Immediately below the sequences are the predicted structures for the \apo{} state, and below those are the predicted structures for the \holo{} state.  Base pairs and unpaired positions are represented by parentheses and full stops, respectively.  Secondary structure and free energy predictions for both states were calculated as described in the Methods section.  (a) In the \apo{} state, the nexus is predicted to base-pair with the aptamer, but in the \holo{} state, the sgRNA is predicted to fold correctly.  (b) The \apo{} and \holo{} states are predicted to have the same fold (consistent with the mechanism proposed in \refsuppfig{mutagenesis}), but note that neither prediction recapitulates the known sgRNA stems very well.

\suppfig{mutagenesis}

    \fignatsize{supp_mat/mutagenesis_abc}

    (Figure continued on next page.)

    \fignatsize{supp_mat/mutagenesis_de}

    \captitle{\ligrnaB{} works by preventing a uracil from burying in Cas9}

    (a) Schematic illustrating the idea that \ligrnaB{} works by sequestering the indicated uracil.  The uracil is always unpaired in wildtype sgRNA (left).  The hypothesis is that the same uracil in \ligrnaB{} is at least occasionally unpaired in the \apo{} state (center), but is always paired in the \holo{} state (right).
    (b) A crystal structure of Cas9 in complex with an sgRNA (4UN3) shows the indicated uracil flipped out and buried in the protein \autocite{nishimasu2014}.
    (c-e) Flow cytometry traces and fold change plots for various mutants of \ligrnaB{}.  The labels give the sequence of the particular mutant being tested.  Any mutations relative to \ligrnaB{} are highlighted in yellow.  The uracil in question is indicated with a small triangle.  All other lines and symbols are described in \reffig{2}.
    (c) Positive and negative controls, and \ligrnaB{} itself.
    (d) Strand-swap mutations for each position along the nexus stem.  None of the mutants are as functional as \ligrnaB{}, but the sensor is tolerant of mutation at positions 2, 4, and 5.  Position 3 was expected to be intolerant due to the importance of the uracil.  We cannot rationalize the intolerance of position 1.  In the context of wildtype sgRNA, any base pair is tolerated at this position, but in the context of \ligrnaB{}, no base pair but \seq{CG} is tolerated (data not shown).
    (e) Modulating the strength of the stem above the uracil has a predictable effect on function.  From top to bottom, the mutants are arranged in the order of increasing base-pairing strength.  Populations shifted to the left indicate stronger activation because the fluorescent reporter was more effective repressed.  Our hypothesis (that \ligrnaB{} works by sequestering the uracil upon ligand binding) predicts that strengthening or weakening the stem above the uracil should decrease or increase activation, respectively.  The clear downward diagonal trend in the populations supports this hypothesis and demonstrates that we can tune the dynamic range of \ligrnaB{} to some extent.  The third and fourth mutants (\seq{AU} and \seq{UA}) may even be useful for applications where strong repression is desired, despite the fact that their dynamic range (10x) is somewhat narrower than that of \ligrnaB{} (15x).

\suppfig{ligand-matrix}

    \fignatsize{supp_mat/ligand_matrix}

    \captitle{ligRNA function in the context of different spacers and ligands} 
    We built versions of each ligRNA with both the theophylline aptamer (purple heading) and the 3-methylxanthine aptamer (magenta heading), then tested them with three different ligands: caffeine (caff), theophylline (theo), and 3-methylxanthine (3mx).  Caffeine is a negative control; it is chemically similar to both theophylline and 3-methylxanthine, but it doesn't interact with either aptamer.  The reported values fold changes are relative to treatment with no ligand (i.e. water-only) and are calculated from the modes of cell populations measured by flow cytometry.  Panels (a-d) show data for the G1, R1, G2, and R2 spacers, respectively.  Note that 3-methylxanthine significantly activates the theophylline aptamer, but that the reverse is much less true, especially for \ligrnaB{}.  None of the ligRNAs were very active with the G2 spacer.  Unfortunately, we cannot rationalize this.  The positive and negative controls for G2 repressed and expressed GFP as expected (data not shown) and the predicted affinities between the G2 spacer and the two aptamers are very weak (\kcalmol{-3.1}, compare to \refsuppfig{affinity-correlation}).

\suppfig{yeast-data}

    Yeast data from the El Samad lab, coming soon...

\suppfig{mammalian-data}

    Mammalian data from the Doudna lab, coming soon...

\printbibliography[title=References]

\end{document}
