\documentclass[10pt,oneside]{article}
\input{preamble}

\begin{document}

\title{Controlling Cas9 with ligand-activated and ligand-deactivated sgRNAs}
\author{Kale Kundert, James Lucas, Kyle E. Watters, Christina M. Fitzsimmons, Benjamin L. Oakes, Christof Fellman, Andrew H. Ng, Ben M. Heineike, David F. Savage, Jennifer A. Doudna, Hana El-Samad, Tanja Kortemme}
\maketitle{}

% A Brief Communication is a more concise format used typically to report a significant improvement to a tried-and-tested method, its modification and adaptation to an important original application, or an important new tool or resource of broad interest for the scientific community. This format typically does not exceed 3 printed pages. Brief Communications begin with a brief unreferenced abstract (3 sentences, no more than 70 words), which will appear on Medline. The title is limited to 10 words (or 90 characters). The main text is typically 1,000-1,500 words, including the abstract and contains no headings with the exception of a single heading for Methods to point readers to the online Methods section providing all technical details necessary for the independent reproduction of the methodology. Brief Communications normally have no more than 2 display items, although this may be flexible at the discretion of the editor, provided the page limit is observed. As a guideline, Brief Communications allow up to 20 references, and article titles are omitted from the reference list.

\section{Abstract}

\section{Introduction}

%¶ Hmmmm...

%¶ Introduction (Fig 1a)

\section{Results}

% The results section needs to convince the reader that the central claim is supported by data and logic. Every scientific argument has its own particular logical structure, which dictates the sequence in which its elements should be presented.

% Each paragraph in the results section starts with a sentence or two that sets up the question that the paragraph answers. The middle of the paragraph presents data and logic that pertain to the question, and the paragraph ends with a sentence that answers the question.

% Central claim: We can control CRISPRi with ligand-activated and ligand-deactivated sgRNAs.

%¶ All three solvent-exposed stems can be used to control the sgRNA (Fig 1b,1d,1e, Supp Tab [rational designs]).  Not enough to simply replace the stem.

We began by quickly screening 97 rational designs using an \invitro{} Cas9 cleavage assay (\refsupptab{rational-designs}) to learn which of the sgRNA stems were most sensitive to the presence of an aptamer, and which strategies for linking the aptamer to the sgRNA were most effective.  

%¶ We did a FACS screen to find linker sequences that would work well (Fig 2a,2b,2c, Supp Fig [in vitro in vivo]).

%¶ Our designs are spacer-independent (Fig 2d, Supp Fig [good-gel]).

%¶ Our designs can be exhibit a remarkably graded response to theophylline  (Fig 2e).  Inflection point is ...

%¶ Our designs are not sensitive to the expression level of the sgRNA (Supp Fig [j23150]).

%¶ We sought to understand the mechanism by which our ligRNAs functioned. (Supp Fig [vienna predictions], [mutagenesis], [shape-seq])

%¶ ligRNA⁻ can be used to control two genes with two different ligands (Fig 3a,3b,3c, Supp Fig [ligand-matrix]).

\section{Discussion}

% The discussion section explains how the results have filled the gap that was identified in the introduction, provides caveats to the interpretation, and describes how the paper advances the field by providing new opportunities.  This is typically done by recapitulating the results, discussing the limitations, and then revealing how the central contribution may catalyze future progress.

%¶ Our ligRNAs are not functional in eukaryotic systems (Supp Fig [mammalian data], [yeast data]).  We believe this is because ....  This is an avenue for further development.

%¶ There have been two other ligand-controlled sgRNA systems described recently.  Compared to these, our system ... (is better).

%¶ We believe that it will be increasingly important to study the function of the spatial and temporal dependencies on genetic circuits, and believe that high-throughput way to regulate gene expression will be critical in the endeavor.

\section{Methods}

\subsection{\Invitro{} Cas9 cleavage assay}

\Invitro{} transcription: Linear, double-stranded template DNA was acquired either by ordering gBlocks® Gene Fragments from IDT (\reffig{1}) or by cloning the desired sequence into a pUC vector and digesting it with with EcoRI and HindIII (\reffig{2}).  Each construct had the following T7 promoter --- \seq{TATAGTAATAATACGACTCACTATAG} --- and a spacer sequence that began with at least 3 G's.  The HiScribe™ T7 High Yield RNA Synthesis Kit (NEB E2040S) was used to transcribe \ngx{10-50} of DNA template, and the RNA Clean \& Concentrator™-25 spin columns (Zymo R1018) were used to remove unincorporated nucleotides.  Target DNA: Target DNA was prepared by using inverse PCR to clone the appropriate target sequence into the pCR2.1 vector roughly across from its XmnI site (i.e.\ in the MCS).  The vector was then digested with XmnI (NEB R0194S) as follows: mix \uL{43.5} \nguL{≈500} miniprepped pCR2.1 DNA, \uL{5.0} 10x CutSmart buffer, and \uL{1.5} \UuL{20} XmnI; incubate at 37°C until no uncleaved plasmid is detectable on a gel (usually \minute{30--60}); dilute to \nM{30}; store at -20°C.  Cas9 reaction: We adapted the following protocol from \namedcite{briner2014}: mix \uL{5.0} water or \mM{30} theophylline and \uL{1.5} \uM{1.5} sgRNA; incubate at 95°C for \minute{3}, then at 4°C for \minute{1}; add \uL{5.48} water, \uL{1.5} 10x Cas9 buffer, and \uL{0.02} \uM{20} Cas9 (NEB M0386T) via master mix; incubate at room temperature for \minute{10}; add \uL{1.5} \nM{30} target DNA; pipet to mix; incubate at 37°C for \hour{1}; add \uL{0.09} \mgmL{20} RNase~A (Sigma R6148), \uL{0.09} \mgmL{20} Proteinase~K (Denville CB3210-5), and \uL{2.82} 6x loading dye via master mix; incubate at 37°C for \minute{20}, then at 55°C for \minute{20}; run the entire reaction (\uL{18}) on a 1\% agarose/TAE/GelRed gel at \Vcm{4.5} for \minute{70}.  Gel quantification: Band intensities were quantified using Fiji (1.51r).  The background was subtracted from each image using a \px{50} rolling ball radius.  The fraction of DNA cleaved in each lane was calculated:

\begin{displaymath}
 \mathrm{f} = \frac{\mathrm{pixels}_\mathrm{2kb}}{\mathrm{pixels}_\mathrm{4kb} + \mathrm{pixels}_\mathrm{2kb}}
\end{displaymath}

The change in cleavage due to ligand was calculated: 

\begin{displaymath}
 \mathrm{Δf} = \mathrm{f}_\mathrm{theo} - \mathrm{f}_\mathrm{apo}
\end{displaymath}

\subsection{CRISPRi assay}

Strain: The strain used for all CRISPRi experiments was \ecoli{} MG1655 with dCas9 and ChlorR on a p15A plasmid (pgRNA-bacteria, Addgene 44251), sgRNA and AmpR on a pUC plasmid (pdCas9-bacteria, Addgene 44249), and sfGFP \autocite{pedelacq2006}, mRFP \autocite{campbell2002}, and KanR chromosomally integrated.  This strain was originally described by \namedcite{qi2013}

Flow cytometry: Overnight cultures were started from freshly picked colonies in \mL{1} LB with \ugmL{100} carbenicillin and \ugmL{35} chloramphenicol.  The next morning, day cultures were started in \mL{15} culture tubes or 24-well blocks by inoculating \uL{4} of saturated overnight culture into \mL{1} EZ Rich Defined Medium (Teknova M2105) with 0.1\% glucose, \ugmL{1} an\-hydro\-tetra\-cycline, \ugmL{100} carbenicillin, \ugmL{35} chloramphenicol, and either \mM{1} theophylline or not.  These cultures were then grown for 8h at 37°C with shaking at \rpm{225} before GFP and RFP fluorescence were measured using a BD LSRII flow cytometer.  Approximately \unit[10,000]{events} were recorded for each measurement.  Biological replicates were done on different days using different colonies from the same transformation.

Data analysis: Cell distributions were obtained by computing a Gaussian kernel density estimation (KDE) over the base-10 logarithms of the measured fluorescence values.  The mode was considered to be the center of each distribution (e.g.\ for determining fold changes) and was obtained through the BFGS maximization of the KDE.

\subsection{FACS screen}

Library cloning: Randomized regions were inserted into the sgRNA using inverse PCR with phosphate-modified and HPLC-purified primers containing degenerate nucleotides.  The PCR and ligation reactions were setup as follows: mix \uL{19.0} water, \uL{2.5} \uM{5} forward primer, \uL{2.5} \uM{5} reverse primer, \uL{1.0} \pguL{100} template DNA and \uL{25.0} Q5® High-Fidelity 2x Master Mix (NEB M0492L); run PCR according to NEB's recommendations, with a \minute{2} extension time and an annealing temperature around 60°C; add \uL{1} \UuL{20} DpnI (NEB R0176L); incubate at 37°C for \hour{1}; purify using QIAquick spin columns (Qiagen 28106) and elute in \uL{50} water; add \uL{5.67} 10x T4 ligase buffer and \uL{1} \UuL{400} T4 ligase (NEB M0202L); incubate overnight at 16°C; load the ligated DNA onto Zymoclean™ spin columns (Zymo D4002) and elute in \uL{10} water to desalt and concentrate it.

Electrotransformation: Electrocompetent cells were prepared as follows: make ``low-salt'' SOB media: \g{20} bacto-tryptone, \g{5} bacto-yeast extract, \mL{2} \molar{5} NaCl, \uL{833.3} \molar{3} KCl, water to 1L, pH to 7.0 with NaOH, autoclave 30 min at 121°C; pick a fresh colony and grow overnight in \mL{1} SOB; in the morning, inoculate \liter{1} SOB with the entire overnight culture; grow at 37°C with shaking at \rpm{225} until OD=0.4 (\hour{≈4}); place cells in an ice bath for 10 min to quickly stop their growth; wash with 400 mL pre-chilled water, then 200 mL pre-chilled water, then 200 mL pre-chilled 10\% glycerol; resuspend in a total volume of \mL{6} pre-chilled 10\% glycerol; make \uL{100} aliquots; flash-freeze and store at -80°C.  Electrocompetent cells were transformed as follows: thaw competent cells on ice for \minute{10}; pipet once to mix cells with \uL{2} \nguL{≈250} library plasmid; shock at \kV{1.8} with a \ms{5} decay time; immediately add \mL{1} pre-warmed SOC; recover at 37°C for \hour{1}; dilute into selective liquid media and grow at 37°C with shaking at \rpm{225} overnight.  After PCR and ligation, libraries were first transformed into electrocompetent Top10 cells, then mira-prepped \autocite{pronobis2016}, sequenced, and combined as necessary, then transformed again into electrocompetent MG1655 cells already harboring the dCas9 plasmid.

Sorting: Cells were grown as for the CRISPRi assay, but when starting new  cultures, care was taken to subculture at least 10x more cells than the size of the library (often \uL{200}).  Sorting was done using a BD FACSAria II cell sorter.  Sorting was no slower than \evts{1000} and no faster than \evts{20,000}, with the slower speeds being more accurate and the faster speeds being necessary to sort large libraries.  Gates were drawn based on the position of the wildtype population if possible, and based on the most extreme library members otherwise.  All gates were drawn diagonally in GFP vs.\ RFP space.  Sorted cells were collected in \mL{1} SOC and, after sorting, were diluted into selective media and grown at 37°C with shaking at \rpm{225} overnight.

Screening for \ligrnaB{}: Libraries are as in \refsupptab{libraries}

\section{Figures}

\fig{1}

 \figonecol{figure_1}

 \captitle{Rational design of ligand-sensitive sgRNAs}
 (a) Schematic illustrating the design strategy.
% * <kortemme@cgl.ucsf.edu> 2017-11-02T12:46:38.023Z:
%
% more details here after Fig revision
%
% ^.
 (b) Aptamer insertion sites \autocite{briner2014}.
% * <kortemme@cgl.ucsf.edu> 2017-11-02T12:47:34.000Z:
%
% insert ref after "domains"
%
% ^.
 (c) Ligand binding causes the ends of the theophylline aptamer to coalesce into a rigid, stem-like structure \autocite{zimmerman1997}.
% * <kortemme@cgl.ucsf.edu> 2017-11-02T12:48:01.000Z:
%
% delete panel c
%
% ^.
 (d) Efficiency of \invitro{} Cas9 cleavage of DNA in the presence and absence of 10mM theophylline for selected ligRNAs  (full data avaliable as Supplemental XXX).  Design numbers refer to \refsupptab{rational-designs} and are color-coded by aptamer insertion site in panel (b).  Percent cut values (bottom) are the average of at least two experiments.  All data shown are from a single gel.
% * <kortemme@cgl.ucsf.edu> 2017-11-02T12:56:29.000Z:
%
% should replace "sgRNA construct" with whatever nomenclature we decide on, i.e. ligRNA- etc
%
% ^.
 (e) Graded cleavage efficiency in response to increasing concentrations of theophylline for a selected sgRNA construct.

\fig{2}

 \figtwocol{figure_2}

 \captitle{Robust ligRNAs identified by \invivo{} screen}
 %
 (a) A schematic of the screen used to isolate \ligrnaF{}.  The numbers above the arrows indicate approximate library sizes at each step.  G1, G2, R1, and R2 refer to different spacers targeting GFP and RFP, respectively.
 (b,c) Single-cell fluorescence distributions for \ligrnaF{} (teal) and \ligrnaB{} (navy) with (solid lines) and without (dashed lines) theophylline.  Control distributions are in grey.  The mode of each distribution is indicated with a plus sign.  Fluorescence values for each cell are normalized by GFP fluorescence for that cell and the modes of the un-repressed control populations measured for that replicate.
 (d) \Invitro{} cleavage data for both ligRNAs in the context of 24 randomly chosen spacers.  The colors indicate the change in the percent of DNA cleaved with and without theophylline.  Each box represents the mean of three measurements with a single spacer.   The grey error bar in the color scale shows the mean and standard deviation for 143 control measurements.
 (e) \Invivo{} theophylline titration for both ligRNAs.  The fluorescence axis is the same as in (b) and (c).
 

\fig{3}

 \figonecol{figure_3}

 \captitle{Parallel control of two genes with two ligands} 
 (a,b) Schematic illustrating the constructs used in the experiment, and the expected consequences of adding theophylline (theo) and 3-methylxanthine (3mx) for each reporter.
 (c) Fluorescence values measured over a \hour{48} timecourse.  Single-cell fluorescence values were measured by flow cytometry.  Bar heights and error bars represent the modes and standard deviations of the cell distributions, respectively.  Unlike in \reffig{2}, fluorescence is normalized by side-scatter (SSC) because both fluorescent channels are being manipulated.

\section{Tables}

\tab{1}

 \tabonecol{table_1/summary_tabular}

 \captitle{Summary of rational design results.}  The results are organized by the design strategy we employed and the domain the sgRNA was inserted into.  We tested 97 designs and found 10 that were active, including at least one active design for each domain.  We considered a design active if it exhibited a >15\% change in cleavage in response to theophylline; 15\% is about the point at which the difference is clearly visible on a gel.

\section{Supplementary Materials}

\supptab{rational-designs}

 \tabtwocol{supp_mat/rational_design_sequences}

 \captitle{Complete results from the \invitro{} screen of rational designs.}  
 \#: The number used to refer to a rational design in the main text.
 Strategy: The mechanism by which the design was intended to work.  "Stem Replacement" means a stem in the sgRNA was simply replaced with by the aptamer (possibly with a linker).  "Induced dimerization" means the sgRNA was slit in half, with each half getting part of the aptamer, in the hope that the two halves would dimerize in the presence of ligand.  "Strand displacement" means that we designed strands that could base pair in two ways --- one with wildtype sgRNA 2° structure and the other somehow different --- in the hope that the aptamer would trigger a switch between the two conformations.
 Domain: Where in the sgRNA the aptamer was inserted.  
 Cleavage: The percent of DNA that was cleaved by a design in the \invitro{} assay.  The \apo{} and \holo{} columns refer to the cleavage with and without theophylline, respectively, the Δ column is the difference between those, and the σ column is the standard deviation of the Δ values for the designs with more than one replicate.  All the percentages are rounded to the nearest 1.
 N: The number of replicates for each design.  The reported percentages are the average of each replicate.
 Active: We considered a design to be acitve if it exhibited a >15\% change in cleavage in response to ligand.  15\% is about the point at which the difference is clearly visible on a gel.
 Sequence: The sequence of the design, including the spacer.

\supptab{libraries}

 \tabtwocol{supp_mat/library_sequences/library_tabular}

 \captitle{Library sequences screened via FACS}

\printbibliography[title=References]

\end{document}
